### DISCUSSION TITLE
How to?

### DISCUSSION THREAD

**Discussion by JohnKesko** [AvaloniaCommunity/Material.Avalonia#275] [Q&A] [answered]
Hi, I'm obviously doing something wrong here, but what?

Using Avalonia 11.02, Material.Avalonia 3.0.0-rc0.99-nightly

In App.axaml:

```
<Application xmlns="https://github.com/avaloniaui"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             x:Class="AvaloniaApplication2.App"
             xmlns:local="using:AvaloniaApplication2"
             xmlns:themes="clr-namespace:Material.Styles.Themes;assembly=Material.Styles">

    <Application.DataTemplates>
        <local:ViewLocator/>
    </Application.DataTemplates>
  
    <Application.Styles>
        <themes:MaterialTheme BaseTheme="Dark" PrimaryColor="Purple" SecondaryColor="Lime" />
    </Application.Styles>
    
</Application>
````

Just trying to compile:
```
App.axaml(12,10): Error AVLN:0004 Avalonia: Unable to find suitable setter or adder for property Styles of type Avalonia.Controls:Avalonia.Application for argument Material.Styles:Material.Styles.Themes.MaterialTheme, available setter parameter lists are:
Avalonia.Styling.IStyle
Avalonia.Styling.IStyle Line 12, position 10.
```

**Comment by SKProCH** [comment, marked as answer]
Hello, @JohnKesko 

How you did manage to find `3.0.0-rc*` version? This is a very outdated package version (named rc because i wasn't think what we should do A LOT of work before actual 3.0.0 release). You should use the latest package, available on nuget. Currently its `3.0.0-avalonia11-preview5.221-nightly`.

**Reply by JohnKesko** [reply]
Ahaa, using the "avalonia11-preview" worked right away! Yeah, I'm using Rider and 3.0.0-rc0.xxx is at the top when searching, so it's easy to assume those nightly builds were the latest (they usually are in the list) but for once I didn't read the release info about the package also.. 🤦‍♂️ 

![Screenshot 2023-08-07 at 22 04 15](https://github.com/AvaloniaCommunity/Material.Avalonia/assets/87022265/3f99187a-9d86-4b6c-8e3e-46127d0e195f)
